Gemfile.lock

```diff
@@ -1,9 +1,8 @@
 PATH
   remote: .
   specs:
-    constantcontact (2.0.1)
+    constantcontact (2.0.1.1)
```

Comment:
Per our standards (http://semver.org/) this should be 2.1.0 as you made API changes that are backwards compatible.